media_softlet/linux/common/vp/ddi/ddi_vp_functions.cpp

```diff
@@ -23,7 +23,9 @@
 //! \file     ddi_vp_functions.cpp
 //! \brief    ddi vp functions implementaion.
 //!
+#if defined(X11_FOUND)
 #include <va/va_dricommon.h>
+#endif
```

Comment:
You could remove this include since header "media_libva_putsurface_linux.h" has been included in line39.

Reply:
Thanks for the reivew ad comments. Updated